pom.xml

```diff
@@ -10,7 +10,7 @@
 
     <groupId>com.hubspot.jackson</groupId>
     <artifactId>jackson-datatype-protobuf</artifactId>
-    <version>0.9.4-SNAPSHOT</version>
+    <version>0.9.4</version>
```

Comment:
Should probably keep the snapshot version, will get changed as part of release process